apps/system-e2e/src/tests/islandis/judicial-system/smoke/first-test.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { BrowserContext, expect, test } from '@playwright/test'
+
+import { urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+
+test.use({ baseURL: urls.islandisBaseUrl })
+
+test.describe('First test', () => {
+  let context: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    context = await session({
+      browser,
+      homeUrl: 'http://localhost:4200/krofur',
+      phoneNumber: '0103019',
+      idsLoginOn: false,
+    })
+  })
+
+  test.afterAll(async () => await context.close())
+
+  test('should work', async () => {
+    const page = await context.newPage()
+    await page.goto('/api/auth/login?nationalId=0000000009')
```

Comment:
This looks strange. Is this how login works in the Judicial System Kröfur?

Reply:
Yeah, this is strange, I should have explained.

We've created a way to bypass the login in our system everywhere but in PROD and that's how we do it.